fbw-a380x/src/systems/instruments/src/MFD/FMC/FmcAircraftInterface.ts

```diff
@@ -209,34 +210,26 @@ export class FmcAircraftInterface {
     if (!this.flightPlanService.hasActive) {
       return;
     }
-    const originTransitionAltitude = this.flightPlanService.active.performanceData.transitionAltitude; // as altitude
-    const destinationTransitionLevel = this.flightPlanService.active.performanceData.transitionLevel ?? 18_000; // as FL
 
     this.arincTransitionAltitude.setBnrValue(
-      originTransitionAltitude !== null ? originTransitionAltitude : 0,
-      originTransitionAltitude !== null
+      this.flightPlanService.active.performanceData.transitionAltitude ?? 2_500, // as altitude
```

Comment:
```suggestion
      this.flightPlanService.active.performanceData.transitionAltitude ?? 0, // as altitude
```
Should be 0/NCD when not entered.